packages/preview3-shim/lib/nodejs/filesystem/descriptor.js

```diff
@@ -0,0 +1,976 @@
+import fs from 'node:fs/promises';
+import process from 'node:process';
+
+import { StreamReader } from '../stream.js';
+import { FsError } from './error.js';
+import { FutureReader } from '../future.js';
+import { ResourceWorker } from '../workers/resource-worker.js';
+import { earlyDispose, registerDispose } from '../finalization.js';
+
+const symbolDispose = Symbol.dispose || Symbol.for('dispose');
+
+const _worker = new ResourceWorker(
+    new URL('../workers/filesystem-worker.js', import.meta.url)
+);
+
+class Descriptor {
+    // Node.js file handle for file operations (FileHandle from fs.promises.open)
```

Comment:
```suggestion
    /** Node.js file handle for file operations (FileHandle from fs.promises.open) */
```
Super nit but IIRC JSDoc has this syntax for single lines